BusyList/Handlers/ListHandler.cs

```diff
@@ -0,0 +1,34 @@
+﻿using BusyList.Commands;
+using System;
+using System.Collections.Generic;
+
+namespace BusyList.Handlers
+{
+    public class ListHandler : IHandler<DoneCommand>
+    {
+        private readonly ITaskRepository _taskRepository;
+
+        public ListHandler(ITaskRepository taskRepository)
+        {
+            _taskRepository = taskRepository;
+        }
+
+        public void Run(DoneCommand command)
+        {
+            IEnumerable<TaskItem> tasks = _taskRepository.GetAll();
+
+            string seperator = new String('-', 10);
+
+            Console.WriteLine("Tasks:");
+
+            foreach (var item in tasks)
+            {
+                Console.WriteLine(seperator);
+                Console.WriteLine($"Task id: {item.Id}");
+                Console.WriteLine($"Description: {item.Description}");
+                Console.WriteLine($"Status: {item.TaskStatus}");
+                Console.WriteLine(seperator);
```

Comment:
The separator will look a bit weird if there are multiple items:

"---------"
text
"---------"
"---------"
text
"---------"

To fix this, I suggest moving the first separator print before the loop, which will create the "expected" output in all cases.